### PR TITLE
coqPackages.mathcomp-analysis: 0.3.9 -> 0.3.10

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 136938` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>coqPackages.mathcomp-analysis</li>
  </ul>
</details>

File: pkgs/development/coq-modules/mathcomp-analysis/default.nix [hunk 1]
```diff
@@ -8,6 +8,7 @@ let mca = mkCoqDerivation {
   pname = "analysis";
   owner = "math-comp";
 
+  release."0.3.10".sha256 = "sha256-FBH2c8QRibq5Ycw/ieB8mZl0fDiPrYdIzZ6W/A3pIhI=";
```
Comment (AndersonTorres) [inline pkgs/development/coq-modules/mathcomp-analysis/default.nix:11]: If the old version is not deleted, the commit message's "title" should be `coqPackages.mathcomp-analysis: init at 0.3.10`, with some explanation such as "including the new version along the older one".

File: pkgs/development/coq-modules/mathcomp-analysis/default.nix [hunk 2]
```diff
@@ -35,8 +36,7 @@ let mca = mkCoqDerivation {
   };
 }; in
 mca.overrideAttrs (o:
-  let ext = { propagatedBuildInputs = o.propagatedBuildInputs
-                                      ++ [ hierarchy-builder ]; };
+  let ext = { propagatedBuildInputs = o.propagatedBuildInputs ++ [ hierarchy-builder ]; };
```
Comment (AndersonTorres) [inline pkgs/development/coq-modules/mathcomp-analysis/default.nix:39]: Does it respect the magic 80-line limit? If not, there is no reason to merge the lines IMHO.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/development/coq-modules/mathcomp-analysis/default.nix (hunk 2 above)
Comment (siraben) [reply pkgs/development/coq-modules/mathcomp-analysis/default.nix:39]: Sure, just wasn't sure what the newline was there for.

File: pkgs/development/coq-modules/mathcomp-analysis/default.nix (hunk 1 above)
Comment (siraben) [reply pkgs/development/coq-modules/mathcomp-analysis/default.nix:11]: According to `git log`, the convention for `coqPackages` is what I have in the PR. It is understood that for this package set, old versions are not discarded.

**Pull request merged**